Application/Interfaces/IUserConfigurationService.cs

```diff
@@ -0,0 +1,28 @@
+using Application.DTOs;
+
+namespace Application.Interfaces;
+
+/// <summary>
+/// Defines a contract related to user configuration.
+/// </summary>
+public interface IUserConfigurationService
+{
+    /// <summary>
+    /// Retrieves a user configuration by its unique identifier.
+    /// </summary>
+    /// <param name="userId">The ID of the user to retrieve configuration for.</param>
+    /// <returns>
+    /// The user configuration if found; otherwise, <c>null</c>.
+    /// </returns>
```

Comment:
To nie zwraca nulla.

```suggestion
    /// <returns>The user configuration.</returns>
```